posawesome/public/js/posapp/components/pos/ItemsSelector.vue

```
@@ -451,123 +469,43 @@ export default {
   },
 
   computed: {
-    filtered_items() {
+  filtered_items() {
       this.search = this.get_search(this.first_search);
```

ESLint

[error] Unexpected side effect in "filtered_items" computed property.

(vue/no-side-effects-in-computed-properties)